notebooks/257-stable-diffusion-torchdynamo-backend/257-stable-diffusion-torchdynamo-backend.ipynb

```diff
@@ -0,0 +1,302 @@
+{
```

Comment:
<p>actually not necessary that order of GPU will be this one. it looks like some misleading info. any system with multiple gpu will have indexes (e.g. if I have not intel GPU, but have 3 cuda gpu, they also will be listed as GPU.0, GPU.1, GPU.2).</p><p> </p><p>We use widgets for selecting devices in notebooks please based on openvino.Core.available_devices info. I think it makes sense to use the same approach here too. Please check other notebooks as examples</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>